### PR TITLE
Add new university UAB

### PR DESCRIPTION
**Pull request by Albert0147**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -19699,6 +19708,7 @@ X. Sean Wang,Fudan University,http://www.cs.fudan.edu.cn/?page_id=1249?lang=eng,
 Xavier Bresson,Nanyang Technological University,http://research.ntu.edu.sg/expertise/academicprofile/Pages/StaffProfile.aspx?ST_EMAILID=XBRESSON&CategoryDescription=Mathematics,9pSK04MAAAAJ
 Xavier Défago,Tokyo Institute of Technology,https://www.coord.c.titech.ac.jp,vPNvIusAAAAJ
 Xavier Le Pallec,CRIStAL,http://cristal.univ-lille.fr/~lepallec,zO9OcTMAAAAJ
+Xavier Otazu,Autonomous University of Barcelona,http://www.cvc.uab.es/~xotazu/,I-U-oUkAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:19711]: Title says "senior scientist", not tenure-track faculty member.

File: csrankings.csv [hunk 2]
```diff
@@ -9187,6 +9190,7 @@ Joon Hee Han,POSTECH,http://cvlab.postech.ac.kr/jhhan,_mMeOTgAAAAJ
 Joos C. A. M. Buijs,TU Eindhoven,https://www.tue.nl/en/university/departments/mathematics-and-computer-science/the-department/staff/detail/ep/e/d/ep-uid/20020361,NOSCHOLARPAGE
 Joost Batenburg,CWI,https://www.cwi.nl/people/1742,6WpX7xUAAAAJ
 Joost Broekens,Leiden University,http://www.joostbroekens.com,vw2QkREAAAAJ
+Joost van de Weijer,Autonomous University of Barcelona,http://www.cvc.uab.es/LAMP/joost/,Gsw2iUEAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:9193]: Senior scientist, not faculty.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please go through the entire list and remove everyone who is not a faculty member.

File: csrankings.csv (hunk 2 above)
Comment (Albert0147) [reply csrankings.csv:9193]: Most of these members belong to the computer vision center (CVC) in UAB, maybe the system in CVC is different. Those members are PIs in different research groups, all can supervise PhD student and postDoc solo.

Comment (emeryberger) [reply csrankings.csv:9193]: It doesn't matter: CSrankings is currently limited to tenure-track faculty.

**Pull request closed without merging**